crates/ty_python_semantic/resources/mdtest/dataclasses/dataclasses.md

```diff
@@ -461,7 +461,51 @@ del frozen.x  # TODO this should emit an [invalid-assignment]
 
 ### `match_args`
```

Comment:
This parameter doesn't exist on 3.9. But maybe not worth worrying about since 3.9 is end-of-life

Reply:
Fixed. Decided not add a test, though.